src/SB/Game/zNPCTypeDutchman.h

```diff
@@ -309,6 +309,7 @@ struct zNPCGoalDutchmanDeath : zNPCGoalCommon
     F32 emit_frac;
     F32 min_y;
     F32 max_y;
+    S32 Enter(float, void*);
```

Comment:
Not trying to be pedantic, but we also want to change `float` to `F32` here in the function definitions/headers files too. We also want to try and keep the parameter names in the headers as well if we know them, for consistency's sake. So adding `dt` and `updCtxt` as well in the headers.